### PR TITLE
x11-misc/polybar: Version bump, fixes GCC 10 build

### PR DESCRIPTION
**Pull request by ajakk**
Bug: https://bugs.gentoo.org/722432
Package-Manager: Portage-2.3.100, Repoman-2.3.22
Signed-off-by: John Helmert III <jchelmert3@posteo.net>

### CODE REVIEW

File: x11-misc/polybar/polybar-3.4.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+SRC_URI="https://github.com/polybar/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/polybar/xpp/archive/${XPP_COMMIT}.tar.gz -> xpp-${XPP_COMMIT}.tar.gz
+	https://github.com/polybar/i3ipcpp/archive/${I3IPCPP_COMMIT}.tar.gz -> i3ipcpp-${I3IPCPP_COMMIT}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="MIT"
+SLOT="0"
+
+IUSE="alsa curl i3wm ipc mpd network pulseaudio"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
```
Comment (SoapGentoo) [inline x11-misc/polybar/polybar-3.4.3.ebuild:22]: make one block:
```
LICENSE
SLOT
KEYWORDS
IUSE
REQUIRED_USE
```

**Comment by ajakk** [comment]
I emailed johu about comaintainership on x11-misc/polybar and x11-wm/i3-gaps a few weeks ago, no response.

File: x11-misc/polybar/polybar-3.4.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+XPP_COMMIT="8c019e6d7fefd2468791bc1cbf90d68ff7c1ba33"
+I3IPCPP_COMMIT="21ce9060ac7c502225fdbd2f200b1cbdd8eca08d"
+inherit cmake-utils python-single-r1
```
Comment (juippis) [inline x11-misc/polybar/polybar-3.4.3.ebuild:9]: *cmake.eclass* with EAPI-7. Also please add a blank line above `inherit`.

File: x11-misc/polybar/polybar-3.4.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+	$(python_gen_cond_dep 'x11-base/xcb-proto[${PYTHON_MULTI_USEDEP}]')
+	x11-libs/cairo[X,xcb(+)]
+	x11-libs/libxcb[xkb]
+	x11-libs/xcb-util-image
+	x11-libs/xcb-util-wm
+	x11-libs/xcb-util-xrm
+	alsa? ( media-libs/alsa-lib )
+	curl? ( net-misc/curl )
+	i3wm? (
+		dev-libs/jsoncpp
```
Comment (juippis) [inline x11-misc/polybar/polybar-3.4.3.ebuild:34]: Does this get linked, since it's used in `DEPEND`? If yes, add a subslot binder `:=`. But be sure.

Comment (ajakk) [reply x11-misc/polybar/polybar-3.4.3.ebuild]: ldd says yes. subslot binder added

**Pull request closed without merging**